taccsite_cms/static/site_cms/css/src/_imports/objects/o-section.css

```diff
@@ -122,44 +132,13 @@ Styleguide Objects.Section
 .o-section--style-light h6 {
   color: var(--global-color-primary--xx-dark);
 }
-.o-section--style-light + .o-section--style-light {
-  border-top: var(--global-border-width--normal) solid
-              var(--global-color-primary--normal);
-}
 
 /* Modifers: Style: (Fake) Endless Background */
 /* FAQ: Allows background to escape `.container` (whether its self or parent) */
-/* SEE: "Tricks" section */
-
-[class*="o-section--style"] {
-  /* To anchor the `position: absolute` pseudo-element */
-  /* To allow `z-index` on children as necessary */
-  position: relative;
+/* FP-1470: Move this to cross-style code when third style is added */
 
-  /* CAVEAT: We MUST maintain intentional horizontal overflow of pseudo-child */
-  overflow-x: visible !important; /* do not allow accidental hide of fake bkgd */
-}
-[class*="o-section--style"] > * {
-  /* To ensure children are above `position: absolute` pseudo-element */
-  position: relative;
-  z-index: 1;
-}
-[class*="o-section--style"]::before {
-  content: '';
-  z-index: 0; /* To ensure real element children are above this */
-
-  position: absolute;
-  display: block;
-  width: 100vw;
-  top: 0;
-  bottom: 0;
-
-  /* To horizontally center */
-  left: 50%;
-  transform: translateX(-50%);
-}
 /* Except for nested sections */
-.o-section .o-section::before { content: none; }
+.o-section .o-section { box-shadow: none; }
```

Comment:
**Explanation.**
For light __and__ dark sections:
- Delete all the cumbersome `::before` solution.
- Unset `box-shadow` (i.e. extended background) for nested sections.

Reply:
Outdated since recent commit. Resolving comment.